xsd/netex_part_1/part1_tacticalPlanning/netex_routingConstraint_support.xsd

```diff
@@ -155,6 +155,11 @@ Rail transport, Roads and Road transport
 			<xsd:enumeration value="mustBoardInZone"/>
 			<xsd:enumeration value="mustAlightInZone"/>
 			<xsd:enumeration value="passThroughUseOnly"/>
+			<xsd:enumeration value="mustPickUpandDropOffToSameStop">
+				<xsd:annotation>
+					<xsd:documentation>Special zones where  vehicles must be returned to the same stop. +v2.0</xsd:documentation>
```

Comment:
This sentense does not make sence. If it is the same stop, it is not a special zone...